cmd/server-mux.go

```diff
@@ -28,6 +28,14 @@ import (
 	"time"
 )
 
+// The value choosen below is longest word choosen
+// from all the http verbs comprising of
+// "PRI", "OPTIONS", "GET", "HEAD", "POST",
+// "PUT", "DELETE", "TRACE", "CONNECT".
+const (
+	longestWord = 7
```

Comment:
defining it as const like this is also fine

another way, maxHTTPVerbLen can also be inited programatically :
```
var maxHTTPVerbLen = func() int {                                                      
        max := 0                                                                       
        for _, i := range append(defaultHTTP1Methods, defaultHTTP2Methods...) {        
                if len(i) > max {                                                      
                        max = len(i)                                                   
                }                                                                      
        }                                                                              
        return max                                                                     
}()
```